bundles/sirix-core/src/main/java/org/sirix/service/json/JsonNumber.java

```diff
@@ -4,27 +4,32 @@
 import java.math.BigInteger;
 
 public class JsonNumber {
+
+    static Number isDecimal(String stringValue){
```

Comment:
isDecimal should be renamed to stringDecimal since it is not a predicate and to match the other function